### PR TITLE
feat: Add Hydration

### PR DESCRIPTION
**Pull request by NunoAlexandre**
For now only enabling Ethereum -> Hydra transfers

### CODE REVIEW

File: app/src/config/registry.ts [hunk 1]
```diff
@@ -390,6 +402,11 @@ export interface Registry {
   chains: Chain[]
   tokens: Token[]
   routes: Route[]
+  // Assets are often uniquely identified by a "asset id" at each chain, making it a chain-dependant value.
+  // The SDKs we use accept the token symbol as the indexing value to work with a given token but some chains
+  // have multiple tokens with the same symbol, in which case we need this map to provide the exact asset id
+  // to disambiguate. The map is Chain id -> Turtle token id -> Local Asset Id.
+  assetId: Map<string, Map<string, string>>
```
Comment (noahjoeris) [inline app/src/config/registry.ts:409]: What do you think of creating types here?
`Map<ChainId, Map<TokenId, LocalAssetId>>`
For me this is way cleaner and more readable.

File: app/src/config/registry.ts [hunk 2]
```diff
@@ -547,3 +581,11 @@ export function rpcConnectionAsHttps(rpc?: string): string {
   if (!rpc) return ''
   return rpc.replace('wss://', 'https://')
 }
+
+export function getAssetId(env: Environment, chainId: string, tokenId: string): string | undefined {
+  return REGISTRY[env].assetId.get(chainId)?.get(tokenId)
+}
+
+export function isAssetHub(chain: Chain): boolean {
```
Comment (Victor-Salomon) [inline app/src/config/registry.ts:589]: I will see if I can use this one anywhere else 🤙

**Review by noahjoeris — APPROVED** [review]
Nice PR @NunoAlexandre ❤️

File: app/src/config/registry.ts (hunk 1 above)
Comment (NunoAlexandre) [reply app/src/config/registry.ts:409]: It's aesthetically more pleasing but the types will just be aliases so there's no real added value imo. But we can add that if you like, albeit that it's already explained in the comment.

Open to it :)

**Review by Victor-Salomon — APPROVED** [review]
LGTM 💥🦾

File: app/src/config/registry.ts (hunk 1 above)
Comment (NunoAlexandre) [reply app/src/config/registry.ts]: Done

**Pull request merged**